From 03150c2ba82182af3105e7c81aa7206c3b0ea479 Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Fri, 10 Mar 2017 16:56:47 +0800
Subject: [PATCH] arm64: dts: rockchip: add linux,pci-domain for PCIe

We need this to ask PCIe bus allocater to always
assign 0 to our root bus isntead of increasing it
, otherwise the hierarchy would be wrong if we unbind
and bind the root port.

Change-Id: I4ada61c89e617c7bccd92c5f9fa3334cae40603e
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
index ede485142847..312840d480c8 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
@@ -1200,6 +1200,7 @@
 			      "hclk", "pm";
 		bus-range = <0x0 0x1>;
 		max-link-speed = <1>;
+		linux,pci-domain = <0>;
 		msi-map = <0x0 &its 0x0 0x1000>;
 		interrupts = <GIC_SPI 49 IRQ_TYPE_LEVEL_HIGH 0>,
 			     <GIC_SPI 50 IRQ_TYPE_LEVEL_HIGH 0>,
-- 
2.35.3

